### DISCUSSION TITLE
bind-toolsonly doesn't work on an IPv6-only network

### DISCUSSION THREAD

**Discussion by zod076** [pauby/ChocoPackages#229] [General]
Hi, I have a laptop that I am testing IPv6 and when I turn off IPv4, the commands dig and/or host doesn't work and displays the following error message:-

```
PS C:\windows\system32\etc> dig aaaa www.google.com
C:\ProgramData\chocolatey\lib\bind-toolsonly\tools\content\dig.exe: parse of C:\WINDOWS\system32\etc\resolv.conf failed

PS C:\windows\system32\etc> cat resolv.conf
nameserver 2606:4700:4700::64
```

I'm running Chocolatey 2.3.0 and bind-toolsonly 9.16.28. What could be the problem ?

**Comment by pauby** [comment]
This is a software issue, not a package one. I don't use the package or the software it contains so I can't help there.

The error indicates there is an issue parsing resolve.conf so it sounds like a configuration issue.

**Reply by zod076** [reply]
You closed this too soon. But I'll reply my findings anyway. It looked like a configuration issue at first, but when I turn on IPv4 via Network Properties in Windows, the dig and host commands work without changing anything in the resolv.conf file. So I assume that the package bind-toolsonly doesn't seem to work when the network is an IPv6-only network.

My question is, is the package bind-toolsonly intended to work only on a dual-stack network or an IPv4-only network ?

**Reply by pauby** [reply]
The question you have is about the software. I can't answer those questions because I don't use the software and I don't support the _software_, only the _package_.

The _package_ extracts the last version of the Windows tools onto the operating system (as stated on the [package page](https://community.chocolatey.org/packages/bind-toolsonly#files)). If you have any question about how it works, what it supports, or a broken configuration, you need to ask in a forum / place that can answer that.

If the _package_ failed to extract the software or was otherwise not putting it into a place you could use it, then that would be a _package_ issue.

With all of that said, I didn't close the issue too early. It comes down to the question being asked of the wrong person and asking in the wrong place, for help with the software.

**Reply by zod076** [reply]
Okay thanks. I'll try the Discord support channel.
